filecoin-proofs/src/api/seal.rs

```diff
@@ -149,18 +155,29 @@ pub fn seal_commit<T: AsRef<Path>>(
 ) -> error::Result<SealCommitOutput> {
     info!("seal_commit:start");
 
-    let SealPreCommitOutput {
-        comm_d,
-        comm_r,
-        p_aux,
-        t_aux,
-    } = pre_commit;
+    let SealPreCommitOutput { comm_d, comm_r } = pre_commit;
 
     ensure!(
         verify_pieces(&comm_d, piece_infos, porep_config.into())?,
         "pieces and comm_d do not match"
     );
 
+    let p_aux = {
+        let mut p_aux_bytes = vec![];
+        let mut f_p_aux = File::open(cache_path.as_ref().join(CacheKey::PAux.to_string()))?;
+        f_p_aux.read_to_end(&mut p_aux_bytes)?;
+
+        deserialize(&p_aux_bytes)
+    }?;
+
+    let t_aux = {
```

Comment:
`t_aux` needs to be deleted after `commit` was successfull

Reply:
We delete a little further down, no?

Reply:
Yep, I see the delete:

```
TemporaryAux::<DefaultTreeHasher, DefaultPieceHasher>::delete(t_aux)?;
```

Reply:
I am just blind :)